src/compile/data/aggregate.ts

```diff
@@ -157,14 +159,18 @@ export class AggregateNode extends DataFlowNode {
     const meas: Measures = {};
 
     for (const s of t.aggregate) {
-      const {op, field, as} = s;
+      const {op, field, as, aggregate_param} = s;
       if (op) {
         if (op === 'count') {
           meas['*'] ??= {};
-          meas['*']['count'] = new Set([as ? as : vgField(s, {forAs: true})]);
+          meas['*']['count'] = {aliases: new Set([as ? as : vgField(s, {forAs: true})])};
         } else {
           meas[field] ??= {};
-          meas[field][op] = new Set([as ? as : vgField(s, {forAs: true})]);
+          meas[field][op] = {aliases: new Set([as ? as : vgField(s, {forAs: true})])};
+
+          if (aggregate_param) {
+            meas[field][op].aggregate_param = aggregate_param;
```

Comment:
Yes, that makes sense. I wonder whether we want to use a similar API for transforms. 

```
{
  "op": {"exponential": 0.5},
  "as": "exp_0.5"
}
```

We often have more concise transform APIs than Vega and this would make the API more consistent between encodings and transforms. 

However, it would be a bit less constant with the API in https://vega.github.io/vega-lite/docs/aggregate.html#argmax

```
    "aggregate": [{
      "op": "argmax",
      "field": "US Gross",
      "as": "argmax_US_Gross"
    }]
```

What do you think?

Reply:
@arvind +1s `"op": {"exponential": 0.5}`

Reply:
Makes sense! I'll update `encoding` level transforms to have `"aggregate": {"exponential": 0.5},` and `view` level transforms to have `"op": {"exponential": 0.5}`.

Reply:
Thank you.

Btw, one thing @arvind brought up is that `aggregate_param ` (which is the term in Vega) can be confused with the param concept in Vega-Lite (which Vega doesn't have (yet)). Just wanted to note that here.